modules/client/src/connext.ts

```diff
@@ -235,7 +262,7 @@ export const connect = async (opts: ClientOptions): Promise<IConnextClient> => {
   // since if the hub never submits you should not continue interacting
   log.debug("Reclaiming pending async transfers");
   // no need to await this if it needs collateral
-  client.reclaimPendingAsyncTransfers();
+  await client.reclaimPendingAsyncTransfers();
```

Comment:
if it needs collateral why would we await these reclaims? would take forever for the client to start up

Reply:
@bohendo found out this was causing issues, but now I'm realizing maybe we shouldn't do this. Although maybe we should wait so that we don't let things get weird if you try to do lots of other things while this is happening?